packages/components/src/theme/index.tsx

```diff
@@ -36,16 +28,14 @@ extend( [ namesPlugin, a11yPlugin ] );
  * };
  * ```
  */
-function Theme( props: WordPressComponentProps< ThemeProps, 'div', true > ) {
-	const { accent } = props;
-	if ( accent && ! colord( accent ).isValid() ) {
-		// eslint-disable-next-line no-console
-		console.warn(
-			`wp.components.Theme: "${ accent }" is not a valid color value for the 'accent' prop.`
-		);
-	}
+function Theme( {
+	accent,
+	background,
+	...props
+}: WordPressComponentProps< ThemeProps, 'div', true > ) {
+	const themeVariables = generateThemeVariables( { accent, background } );
```

Comment:
We should probably memoize the theme variables computation, I guess it can be a pretty expensive operation to compute every re-render.

We should probably profile it anyway to see how long it currently takes ?

Reply:
It isn't particularly expensive but yes I think we should memoize it, since `Theme` will usually be a root-ish node than a leaf-ish node. 5db9b4b6ed2effdbedf6062f7cf4f11e99eb1576